File: Javascript/Javascript Fundamentals/Functions/appearance-count.js
function solve(args) {
    var n = +args[0],
    arr = args[1].split(' '),
    number = +args[2];

    console.log(AppearanceCount(number, arr));

    function AppearanceCount(number, array) {
        var count = 0,
        i;

        for(i = 0; i < n; i += 1) {
            if(+array[i] === number){
                count += 1;
            }
        }

        return count;
    }
}

File: C#/C# Advanced/Methods/04_AppearanceCount/AppearanceCount.cs
using System;
using System.Linq;

class AppearanceCount
{
    static void Main()
    {
        int arrSize = int.Parse(Console.ReadLine());
        int[] numbers = new int[arrSize];

        numbers = Console.ReadLine().Split(new char[] { ' ' },
            StringSplitOptions.RemoveEmptyEntries).Select(int.Parse).ToArray();

        int numberToSearch = int.Parse(Console.ReadLine());

        Console.WriteLine(CountAppearances(numbers, numberToSearch));
    }

    static int CountAppearances(int[] arr, int numberForSearching)
    {
        int count = 0;

        foreach (var item in arr)
        {
            if (item == numberForSearching)
            {
                count++;
            }
        }

        return count;
    }
}